### PR TITLE
Add bower.json manifest file

### PR DESCRIPTION
**Pull request by amayvs**
Hi Brian,

I've added a minimal `bower.json` file and bumped the patch version.
I can extend it further as per #136 if required.

Cheers.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "pep",
   "title": "jquery.pep.js",
   "description": "Kinetic drag for mobile & desktop",
-  "version": "0.6.7",
+  "version": "0.6.8",
```
Comment (seanCodes) [inline package.json]: Update to `0.6.11`

File: bower.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "name": "pep",
+  "version": "0.6.8",
```
Comment (seanCodes) [inline bower.json]: Update to `0.6.11`

**Comment by Anaphase** [comment]
For what it's worth, I think package versions should never be changed in Pull Requests. That should be up to the authors to decide when releases are done, especially since multiple PRs might be combined into one release.

**Comment by amayvs** [comment]
@Anaphase Fair point. The reasoning behind it was to tie it to version in `bower.json`.

I no longer have access to old repo. I'll create a new PR. Thanks.

**Comment by seanCodes** [comment]
@Anaphase That sounds good. We should remove the version-bump steps from the contributing doc

**Pull request closed without merging**